packages/pigeon/lib/objc_generator.dart

```diff
@@ -62,48 +62,64 @@ String _className(String? prefix, String className) {
   }
 }
 
-String _callbackForType(TypeDeclaration type, String objcType) {
+String _callbackForType(TypeDeclaration type, _ObjcPtr objcType) {
   return type.isVoid
       ? 'void(^)(NSError *_Nullable)'
-      : 'void(^)($objcType *, NSError *_Nullable)';
+      : 'void(^)(${objcType.ptr}, NSError *_Nullable)';
```

Comment:
The way you've coded the helper, there will be a space before the `,` here, which will look very strange. Perhaps you should have different stringifying methods for different contexts (type only vs. type+name).

Reply:
Yea I looked at these formatting issues while I was developing this.  I've never guaranteed that generated output would match our style guide.  I've just done my best and when it required special logic to make it match I've avoided adding extra complexity to the generator.  Users can always run their generated code through the formatter for their choice, so perfect formatting isn't worth the cost of added complexity at this stage of the project.